wagtailpolls/views/vote.py

```diff
@@ -27,7 +28,7 @@ def vote(request, poll_pk):
         int(poll_pk)
 
     except ValueError:
-        return HttpResponse("<h1>Oops, there is no poll to vote on!</h1>", status=500)
+        return HttpResponse("<h1>{0}</h1>".format(_('Oops, there is no poll to vote on!')), status=500)
```

Comment:
Error messages do not generally get translated. I would suggest this is better off as `raise Http404` as well, although that change is unrelated to translations.